.github/workflows/test-functional-lite.yml

```diff
@@ -49,6 +49,8 @@ jobs:
           always_install_pnpm: true
           build_lite: true
           skip_build: true
+          python_version: "3.10"
+          test: true
```

Comment:
@pngwn I added `test: true` to several invocations of "gradio-app/gradio/.github/actions/install-all-deps@main". The action was failing for me otherwise -- not sure how they were passing before